keps/sig-network/1672-tracking-terminating-endpoints/README.md

```diff
@@ -148,9 +155,123 @@ of the [EndpointSlice API work](/keps/sig-network/20190603-endpointslices/README
 Since this is an addition to the EndpointSlice API, the version skew strategy will follow that
 of the [EndpointSlice API work](/keps/sig-network/20190603-endpointslices/README.md).
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [X] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: EndpointSliceTerminatingCondition
+  - Components depending on the feature gate: kube-apiserver and kube-controller-manager
+
+###### Does enabling the feature change any default behavior?
+
+Yes, terminating endpoints are now included as part of EndpointSlice API. The `ready` condition of an EndpointSlice will always be `false` to ensure consumers do not send traffic to terminating endpoints unless new conditions are checked.
```

Comment:
I'm assuming that you mean `ready` condition of an individual endpoint, not endpoint slice, right?

Maybe something like:
```
Yes, terminating endpoints are now included as part of EndpointSlice API.
However, for each terminating endpoint (`terminating` condition set to true), the `ready` condition will be set to false
to ensure consumer do not send traffic ...
```

?

Reply:
yes, will update